src/CarbonAware.CLI/src/Commands/EmissionsForecasts/LocalizableStrings.resx

```diff
@@ -0,0 +1,135 @@
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="DataEndAtDecsription" xml:space="preserve">
+    <value>End time boundary of forecasted data points. Ignores current forecast data points after this time.</value>
+  </data>
+  <data name="RequestedAtDescription" xml:space="preserve">
+    <value>The timestamp used to access the most recently generated forecast as of that time. If not provided, the current forecast is requested.</value>
+  </data>
+  <data name="DataStartAtDescription" xml:space="preserve">
+    <value>Start time boundary of forecasted data points. Ignores current forecast data points before this time.</value>
+  </data>
+  <data name="WindowSizeDescription" xml:space="preserve">
+    <value>The estimated duration (in minutes) of the workload.</value>
+  </data>
+  <data name="EmissionsForecastDescription" xml:space="preserve">
+    <value>Command keyword to get Forecast data based on location</value>
```

Comment:
The way this shows up in the help text is more a description of what the command does, rather than a description of the keyword.  Consider:

```suggestion
    <value>Gets emissions forecast data for given locations. </value>
```